templates/outer-loop/maven/Dockerfile

```diff
@@ -1,6 +1,8 @@
 # Step 1: Build the user's application
 FROM {{.STACK_IMAGE_MAVEN}} as compile
 
+ENV WLP_INSTALL_PATH={{.WLP_INSTALL_PATH}}
```

Comment:
Seems like more of an ARG not ENV since we're not trying to make it visible in the container.